applications/nrf5340_audio/boards/nrf5340_audio_dk_nrf5340_cpuapp_fota.overlay

```diff
@@ -1,3 +1,5 @@
+#include "nrf5340_audio_dk_nrf5340_cpuapp.overlay"
```

Comment:
Why is this required now? There seems to be no other changes to this file.

Reply:
Will this build without the inclusion of the overlay?

Reply:
Yes it seems to as long as the default for these three items are marked with "n".

Reply:
Discuss.

Reply:
We talked on this as it gives only one place where the LEDs are defined!

Reply:
This wont work as the image built has the required features and applications to perform firmware upgrades.